functions.php

```diff
@@ -38,7 +38,7 @@ function pg_enqueue_site_files() {
 
 		wp_enqueue_style( 'site-fonts', get_template_directory_uri() . '/fonts.min.css', NULL, filemtime( get_template_directory() . '/fonts.css' ) );
 		if ( 'local' === wp_get_environment_type() ) {
-			wp_enqueue_style( 'firebird-styles', 'https://hathitrust-firebird-common.netlify.app/assets/main.css');
+			wp_enqueue_style( 'firebird-styles', 'https://hathitrust-firebird-common.netlify.app/assets/index.css');

```

Comment:
Unrelated change?

Reply:
I originally added the netlify styles/scripts for my local development, but the filename changed while I was out. I guess it's not really related to this ticket, but it needs to be changed in order for my local wordpress theme to function.